test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/PropertiesTest.scala

```diff
@@ -173,4 +175,286 @@ class PropertiesTest {
     prop2.put("1", 1.asInstanceOf[AnyRef])
     assertCollSameElementsAsSet("a", "b", "c", "d")(prop2.stringPropertyNames())
   }
+
+  // ported from Harmony
+
+  @Test def nonStringValues(): Unit = {
+    assumeTrue("Assumed compliant asInstanceOf", hasCompliantAsInstanceOfs)
+    val properties = new Properties
+
+    properties.put("age", new Integer(18))
+    assertNull(properties.getProperty("age"))
+    assertThrows(classOf[ClassCastException],
+                 properties.list(new PrintWriter(new ByteArrayOutputStream())))
```

Comment:
I want to know this too, but based upon my current understanding (https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#indentation) this should be 4 lines indented as a continuation line)

Reply:
Ok, I think I got them all. Thanks.